astroid/inference.py

```diff
@@ -1139,10 +1143,12 @@ def infer_empty_node(
 nodes.EmptyNode._infer = infer_empty_node  # type: ignore[assignment]
 
 
-def _populate_context_lookup(call, context):
+def _populate_context_lookup(
+    call: nodes.Call, context: InferenceContext | None
+) -> dict[nodes.NodeNG, InferenceContext]:
```

Comment:
This feels like it should be `InferenceResult` or something.

Reply:
Which part?

Reply:
The `nodes.NodeNG`